unittests/builtinhttp/builtinhttp.cpp

```diff
@@ -426,6 +426,38 @@ private Q_SLOTS:
         }
     }
 
+    // Test that that if the setUseWSAddressing is set int the client  then the SOAP action
+    // is really placed to the SOAP envelope's header and not embedded to the  Content-Type HTTP header
```

Comment:
I'd say `to` -> `in` both times

two spaces before Content-Type

Reply:
Fixed